### ISSUE TITLE
1.28.10 still depends on DejaVuSans.ttf

### ISSUE THREAD

**Issue by prlw1** [OpenPrinting/cups-filters#411]
According to `NEWS`
```
CHANGES IN V1.28.10
...
        - libfontembed: Removed unneeded fontembed/main.c and ttfread
          executable. Eliminates the dependency on DejaVuSans.ttf
          (Issue #386).
```
but `configure.ac` still contains
```
AS_IF([test "x$cross_compiling" != "xyes" && ! test -f "$with_test_font_path"],
        [AC_MSG_ERROR(DejaVuSans.ttf font file is missing. Please install a package providing it.)]
)
```
so cups-filter 1.28.10 can't be built without DejaVuSans.ttf.

**Comment by khatri-deepak** [comment]
@tillkamppeter @prlw1 I tried to install cupsfilter without dejavu and it shows this error.

![err](https://user-images.githubusercontent.com/55032229/147910804-6a6e576f-72b2-4250-8342-aaaa586148bc.jpeg)

After that, I commented the code for testing dejavu dependecy in configure.ac. 

![codechange](https://user-images.githubusercontent.com/55032229/147910849-9b9186ec-e56f-49f2-bbb3-1a24aeafad27.jpeg)
After that, it compiled correctly without showing any further error.
![correct](https://user-images.githubusercontent.com/55032229/147910903-61444ae7-4386-4446-8010-a212f432a789.jpeg)

So I believe that the above code was mistakenly left behind while deleting the dependency of dejavu, which can be fixed by just removing the above commented code.

So, if you give me permission, I can make the necessary adjustments to the code.

**Comment by tillkamppeter** [comment]
There are 4 test programs which use the test font, but for running cups-filters the test font is not needed.

So do not remove the lines from `configure.ac`, simply make the failure non-fatal, by replacing `AC_MSG_ERROR` by a non-fatal message output.

Fix needs to get applied in both master and 1.x branches of cups-filters.

The location of the test font is put into the `TESTFONT` variable in `config.h` which is used by 4 test programs, 3 of them used by libfontembed:
```
$ grep TESTFONT *
config.h:#define TESTFONT "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf"
config.h.in:#undef TESTFONT
config.log:#define TESTFONT "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf"
config.status:D["TESTFONT"]=" \"/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf\""
configure:#define TESTFONT "$with_test_font_path"
configure.ac:AC_DEFINE_UNQUOTED([TESTFONT], ["$with_test_font_path"], [Path to font used in tests])
$ grep TESTFONT */*
[...]
filter/test_pdf2.c:  const char *fn=TESTFONT;
filter/test_pdf2.c:    printf("Font %s was not loaded, exiting.\n", TESTFONT);
fontembed/test_analyze.c:  const char *fn=TESTFONT;
fontembed/test_analyze.c:    printf("Font %s was not loaded, exiting.\n", TESTFONT);
fontembed/test_pdf.c:  const char *fn=TESTFONT;
fontembed/test_pdf.c:    printf("Font %s was not loaded, exiting.\n", TESTFONT);
fontembed/test_ps.c:  const char *fn=TESTFONT;
fontembed/test_ps.c:    printf("Font %s was not loaded, exiting.\n", TESTFONT);
```

**Comment by khatri-deepak** [comment]
@tillkamppeter  could you please review my PRs?

**Comment by tillkamppeter** [comment]
Fix is also included in today's 1.28.11 release of cups-filters.
